templates/components/accordion.html.twig

```diff
@@ -0,0 +1,31 @@
+{% macro accordion(items, options) %}
+    {% set _id = options.id ?? ('accordion-' ~ random(0, 999)) %}
+    {% set _raw = options.raw ?? true %}
+    {% set _flush = options.flush ?? false %}
+    {% set _always_open = options.always_open ?? false %}
+    {% set _extraClass = options.extraClass ?? '' %}
+
+    <div id="{{ _id }}" class="accordion {% if _flush == true %}accordion-flush{% endif %} {{ _extraClass }}">
+        {% for item in items %}
+            {% set _id_collapse = _id ~ '-collapse-' ~ loop.index %}
+            {% set _item_extraClass = item.options.extraClass ?? '' %}
+            {% set _item_title_extraClass = item.options.title_extraClass ?? '' %}
+            {% set _item_body_extraClass = item.options.body_extraClass ?? '' %}
+            <div class="accordion-item {{ _item_extraClass }}">
+                <div class="accordion-header {{ _item_title_extraClass }}" id="{{ _id }}-heading-{{ loop.index }}">
+                    <button class="accordion-button {% if loop.first == false %}collapsed{% endif %}"
```

Comment:
`is item open` should be an item option, not if not first loop.

Example, I want to have the second accordion item to be opened when arriving in the page.